pkg/R/DataFrame.R

```diff
@@ -114,20 +114,192 @@ setMethod("count",
             callJMethod(sdf, "count")
           })
 
-#' Collect elements of a DataFrame
-#' 
-#' Returns a list of Row objects from a DataFrame
-#' 
-#' @param df A SparkSQL DataFrame
-#' 
+# Collect the rows of a Spark DataFrame and return a named list for each row.
```

Comment:
More explicit: `Collects a Spark DataFrame and returns a list of named lists (each of which represents a row).`